### PR TITLE
French locale changes

### PR DESCRIPTION
**Pull request by GabCoolDude**
Adds french translations

### CODE REVIEW

File: Whisky/Localizable.xcstrings [hunk 1]
```diff
@@ -8651,8 +8651,8 @@
         },
         "fr" : {
           "stringUnit" : {
-            "state" : "needs_review",
-            "value" : "\"%@\" is already pinned!"
+            "state" : "translated",
+            "value" : "\"%@\" est déjà épingler!"
```
Comment (mhammerc) [inline Whisky/Localizable.xcstrings:8655]:
```suggestion
            "value" : "\"%@\" est déjà épinglé !"
```

File: Whisky/Localizable.xcstrings [hunk 2]
```diff
@@ -9005,8 +9005,8 @@
         },
         "fr" : {
           "stringUnit" : {
-            "state" : "needs_review",
-            "value" : "Pin name:"
+            "state" : "translated",
+            "value" : "Nom de l'épingle:"
```
Comment (mhammerc) [inline Whisky/Localizable.xcstrings:9009]:
```suggestion
            "value" : "Nom de l’épingle :"
```

File: Whisky/Localizable.xcstrings [hunk 3]
```diff
@@ -8887,8 +8887,8 @@
         },
         "fr" : {
           "stringUnit" : {
-            "state" : "needs_review",
-            "value" : "Pin Program"
+            "state" : "translated",
+            "value" : "Épingler Programme"
```
Comment (mhammerc) [inline Whisky/Localizable.xcstrings:8891]:
```suggestion
            "value" : "Épingler le Programme"
```

File: Whisky/Localizable.xcstrings [hunk 4]
```diff
@@ -9123,8 +9123,8 @@
         },
         "fr" : {
           "stringUnit" : {
-            "state" : "needs_review",
-            "value" : "Program path:"
+            "state" : "translated",
+            "value" : "Emplacement du programme:"
```
Comment (mhammerc) [inline Whisky/Localizable.xcstrings:9127]:
```suggestion
            "value" : "Emplacement du programme :"
```

File: Whisky/Localizable.xcstrings [hunk 5]
```diff
@@ -9241,8 +9241,8 @@
         },
         "fr" : {
           "stringUnit" : {
-            "state" : "needs_review",
-            "value" : "Pin Program"
+            "state" : "translated",
+            "value" : "Épingler Programme"
```
Comment (mhammerc) [inline Whisky/Localizable.xcstrings:9245]:
```suggestion
            "value" : "Épingler le Programme"
```

**Review by mhammerc — COMMENTED** [review]
Just a few syntax rules, verbs rules, and smart quotes (https://smartquotesforsmartpeople.com/).

This review do not fix any critical errors!

**Review by IsaacMarovitz — APPROVED** [review]

**Comment by IsaacMarovitz** [comment]
Merci beaucoup!

**Pull request merged**